git/history.py

```diff
@@ -17,8 +17,9 @@ def run(self, edit):
         line_ranges = [self.get_lines(selection) for selection in self.view.sel() if not selection.empty()]
 
         if line_ranges:
-            for line_range in line_ranges:
-                command.extend(('-L', str(line_range[0]) + ',' + str(line_range[1])))
+            for (line_range_0, line_range_1, *line_range_len) in line_ranges:
```

Comment:
If we're unpacking it, how about better descriptive names? `range_start` and `range_end`, for instance, would better explain what's happening here.

Reply:
Yes, the better descriptive names would be better explain what happening here.